model/messaging-common.yaml

```diff
@@ -18,6 +18,8 @@ groups:
       - ref: server.port
       - ref: network.protocol.name
         examples: ['amqp', 'mqtt']
+        requirement_level:
+          conditionally_required: Only for messaging systems and frameworks that support more than one protocol.
         tag: connection-level
       - ref: network.protocol.version
```

Comment:
Shouldn't `network.protocol.version` follow the same? Given we add the protocol name, is the version also important together?

Reply:
Discussed in the messaging workgroup meeting.

For now we want to proceed with the changes proposed in this PR to align the use of `network.protocol.*` attributes with HTTP. However, we'll exclude those attributes from initial stability, they will remain experimental and requirement levels can be revisited.